.luacheckrc

```diff
@@ -1,8 +1,9 @@
 unused_args = false
 allow_defined_top = true
+max_line_length = 999
```

Comment:
Why 999 and not 120 (which my VSCode plugin may have been set to check for)?

Reply:
`999` is enough if you're going to have long lines.
It's similar to having unlimited lines.

Reply:
Ok, I guess that's fine then